staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -338,6 +338,8 @@ func TestResultIntoWithNoBodyReturnsErr(t *testing.T) {
 
 func TestURLTemplate(t *testing.T) {
 	uri, _ := url.Parse("http://localhost/some/base/url/path")
+	uriSingleSlash, _ := url.Parse("http://localhost/")
+	uriBaseURL, _ := url.Parse("http://localhost/base/url")
```

Comment:
`uriBaseURL` and `uri` are the same test case. Could you remove `uriBaseURL`?

(I probably asked for a relative path test case (omit leading slash). Let's exclude that topic from this PR)